ambry-tools/src/main/java/com.github.ambry/store/DumpIndexTool.java

```diff
@@ -64,19 +65,25 @@
   private final boolean activeBlobsOnly;
   // True if blob stats needs to be logged, false otherwise
   private final boolean logBlobStats;
+  // set to true if only error logging is required
+  private final boolean silent;
```

Comment:
what is the difference b/w `logBlobStats` and `silent`? Having too many of this gets confusing.